### DISCUSSION TITLE
windows10防火墙配置问题

### DISCUSSION THREAD

**Discussion by max08** [jeessy2/ddns-go#1226] [Q&A]
我是win10电脑装的DDNSGO，已经把本程序加到防火墙白名单里了。在外网ping域名是可以解析出相应的IPV6地址的，但是是不通的。如果把win10的防火墙关掉就能ping通了。请问有人知道怎么解决吗。

**Comment by okrc** [comment]
这些问题建议先尝试向生成式AI提问，这样往往可以更快获得解答。
powershell 以管理员运行
``` powershell
New-NetFirewallRule -DisplayName IPv6-ICMP -Direction Inbound -Protocol ICMPv6
```
